README.md

```diff
@@ -13,53 +13,49 @@ The purpose of this Kubernetes controller is to sync ARM VM/VMSS tags and node l
-    not be deleted when the corresponding tags are deleted, even if `syncDirection` is set to `arm-to-node`.
-    5. `resourceGroupFilter`: The controller can be limited to run on only nodes within a resource group filter (i.e. nodes that exist in RG1, RG2, RG3).
-    Default is `none` for no filter. Otherwise, use name of (single) resource group.
-    6. `minSyncPeriod`: The minimum interval between updates to a node, in a format accepted by golang time library for Duration. Decimal numbers followed by
-    time unit suffix. Valid time units are "ns", "us", "ms", "s", "m", "h". Ex: "300ms", "1.5h", or "2h45m". It may take one default period (5m) for this
-    to update.
-3. You can edit [`config/manager/manager.yaml`](https://github.com/Azure/node-label-operator/blob/master/config/manager/manager.yaml). `sync-period` is the maximum time between calls to reconcile. The default is "10h".
-4. Running the operator:
-    1. To run the controller locally, run `make` to build the controller, then `make run` to run the controller on your cluster.
-    2. To deploy the controller in your cluster, make sure IMG is set (for example, "<dockerhub-username>/node-label-manager") and run `make docker-build docker-push` and `make deploy`.
-
-For a general idea of how to set up a cluster from scratch with this operator installed, see the commands used for setting up test clusters with
-[AKS](https://github.com/Azure/node-label-operator/blob/master/tests/aks/setup.sh) and [aks-engine](https://github.com/Azure/node-label-operator/blob/master/tests/aks-engine/setup.sh).
+## Getting Started 
+
+1. Assume you already have a cluster.
+
+You have created a cluster using either AKS or aks-engine.
+
+2. Create a managed service identity if you don't have one. If you have an AKS cluster, then you will use the MC\_ resource group.
+
+```sh
+export AZURE_RESOURCE_GROUP=<resource-group>
+export AZURE_IDENTITY_LOCATION=~/identity.json
+export AZURE_IDENTITY=<identity-name>
+
+az identity create -g $AZURE_RESOURCE_GROUP -n ${AZURE_IDENTITY} -o json > $AZURE_IDENTITY_LOCATION
+
+export AZURE_IDENTITY_RESOURCE_ID=$(cat ${AZURE_IDENTITY_LOCATION} | jq -r .id)
+export AZURE_IDENTITY_CLIENT_ID=$(cat ${AZURE_IDENTITY_LOCATION} | jq -r .clientId)
+export AZURE_IDENTITY_PRINCIPAL_ID=$(cat ${AZURE_IDENTITY_LOCATION} | jq -r .principalId)
+```
+
+3. Create roles for identity.
+
+```sh
+az role assignment create --role "Managed Identity Operator" --assignee $AZURE_IDENTITY_PRINCIPAL_ID --scope $AZURE_IDENTITY_RESOURCE_ID
+az role assignment create --role "Contributor" --assignee $AZURE_IDENTITY_PRINCIPAL_ID --scope /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${AZURE_RESOURCE_GROUP}
+```
+
+4. Create k8s resources, including controller deployment. 
+
+Clone this repository. From the root directory of the repository, run:
+
+```sh
+make quickstart
```

Comment:
If you use `envsubst` you can just say `cat  config/quickstart/quickstart.yaml | envsubst | kubectl apply -f -` here.

Reply:
I like that it's easier to see what's happening that way.